_data/sites.json

```
@@ -5264,6 +5323,18 @@
         ]
     },
 
+    {
+        "name": "Mongo Atlas",
+        "url": "https://docs.atlas.mongodb.com/tutorial/delete-atlas-account/",
+        "difficulty": "hard",
+        "notes": "To delete your MongoDB account you first need to delete all active clusters, projects and organizations linked with your account (<a href=\"https://docs.atlas.mongodb.com/tutorial/delete-atlas-account/\">Instructions here</a>). Then you can send a mail to cloud-account-support@mongodb.com, with the subject line \"Request for Atlas Account Deletion\" from your regitered mail and delete your account",
```

```suggestion
        "notes": "To delete your MongoDB account you first need to delete all active clusters, projects and organizations linked with your account. Then you can send a mail to cloud-account-support@mongodb.com, with the subject line \"Request for Atlas Account Deletion\" from your registered mail and delete your account",
```
I suggest removing the link from the text as it is already the link for the entry